src/main/java/use_case/TerminalOperations/TerminalOperations.java

```diff
@@ -8,9 +8,11 @@
 
```

Comment:
error handling is not done at where are the errors that are getting thrown being dealt with?